### PR TITLE
fix(contentblockcards): content spacing fix

### PR DESCRIPTION
**Pull request by greeshmareddyg**
### Related Ticket(s)

https://github.com/carbon-design-system/ibm-dotcom-library/issues/2417

### Changelog

**Changed**

- Patterns: content blockcards- Spacing fixes

referred this: https://ibm.ent.box.com/file/657581379075

### CODE REVIEW

File: packages/styles/scss/patterns/blocks/content-block-cards/_content-block-cards.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        padding-bottom: $carbon--layout-06;
+      }
+      @include carbon--breakpoint('lg') {
+        @include contentblockcards-large-spacing;
+      }
+      @include carbon--breakpoint('xlg') {
+        @include contentblockcards-large-spacing;
+      }
+      @include carbon--breakpoint('max') {
+        @include contentblockcards-large-spacing;
```
Comment (annawen1) [inline packages/styles/scss/patterns/blocks/content-block-cards/_content-block-cards.scss:35]: hmm have a feeling we don't need to specify all these breakpoints

Comment (annawen1) [reply packages/styles/scss/patterns/blocks/content-block-cards/_content-block-cards.scss:35]: I believe having this

```
@include carbon--breakpoint('lg') {
        @include contentblockcards-large-spacing;
}
```
should be good 

`carbon--breakpoint('lg')` is the same as `@media (min-width: (whatever rem lg breakpoint is))`

Comment (jeffchew) [reply packages/styles/scss/patterns/blocks/content-block-cards/_content-block-cards.scss:35]: @greeshmareddyg can you take a look?

Comment (greeshmareddyg) [reply packages/styles/scss/patterns/blocks/content-block-cards/_content-block-cards.scss]: @annawen1 @jeffchew applying only lg breakpoint now..

**Pull request merged**